scripts/install.bat

```diff
@@ -1,25 +1,52 @@
-REM usage: install.bat USER PASSWORD
-REM installs/configures the opensafely service
-set directory="E:\job-runner"
-nssm stop opensafely
-nssm remove opensafely confirm
-nssm install opensafely "C:\Program Files\Git\usr\bin\bash" "%directory%\scripts\run.sh"
-nssm set opensafely AppDirectory %directory%
-nssm set opensafely DisplayName "OpenSAFELY job runner"
-nssm set opensafely Start SERVICE_AUTO_START
-REM login as user
-nssm set opensafely ObjectName %1 %2
-nssm set opensafely DependOnService com.docker.service
-nssm set opensafely AppThrottle 10000
-nssm set opensafely AppExit Default Restart
-nssm set opensafely AppRestartDelay 1000
+@echo off
+REM This scripts installs/configures the opensafely service, designed to be
+REM installed on the opensafely TPP backend
+
+set directory=E:\job-runner
+set nssm=C:\nssm-2.24\win64\nssm.exe
+
+if "%1" == "" GOTO :usage
+if "%2" == "" GOTO :usage
```

Comment:
Can you put a comment before these two lines to explain that they're checking both arguments exist?  (assuming that's what it does!)

Reply:
Beyond the fact that we require both arguments?

Reply:
Yep, just to make it clear that's what we're enforcing here.  I've approached this in terms of "if Simon is not around and I have to debug this, which lines made me pause to consider what they do", so having an idea that arguments 1&2 are necessary is really useful.

Reply:
Done